### ISSUE TITLE
Image Block: not able to replace image

### ISSUE THREAD

**Issue by davipontesblog** [Automattic/wp-calypso#37370]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/block-editor/page/domain/page-ID from a home page template page auto-loaded from template-first themes. In this example it was Balasana.
2. Select an image in an Image Block, click on image
3. Click on "Edit Image"
4. Click on "Media Library"
5. Select an image from media library, click "Insert"

#### What I expected
Image to be replaced with the new one I picked

#### What happened instead
Image is not replaced. See post below for more details with recording showing the issue. I was not able to replicate this on a test site.
https://wp.me/p58i-8fZ

**Comment by alaczek** [comment]
I'm wondering if this wasn't a one-off issue. Looks like a few changes were rolled out to the editor since this ticket was opened, and we didn't get more reports of this issue (I was not able to replicate either), so maybe we could close it for now?

**Comment by rachelwinspear** [comment]
I experienced this issue today during a Quick Start session: #3142192-zen
User has switched from Hever to Balasana on Personal Plan.

Recording of the issue: https://d.pr/v/ms3J09

User experienced the issue on Chrome, however, my video was recorded on Firefox.

I was also able to reproduce issue on a test site I set up, Personal plan with Balasana theme.

**Comment by noahtallen** [comment]
Here's what I tried:
1. Created a site via wordpress.com/new
2. Purchased a personal plan along the way
3. Tested that adding an image from the media library worked with the image block.
4. Switched to the balasana theme, and made sure to activate the balasana homepage.
5. Tested that adding an image from the media library worked with the image block. (both for new and existing blocks on the site's homepage)

Feel free to continue reporting if it pops up again, but I'm going to close this for now!
